histogram/src/standard.rs

```diff
@@ -25,11 +23,23 @@ impl Histogram {
 
         Self {
             config: *config,
-            start: SystemTime::now(),
             buckets,
         }
     }
 
+    /// Creates a new histogram using a provided [`crate::Config`] and the
+    /// provided collection of buckets.
+    pub fn from_buckets(config: &Config, buckets: Vec<u64>) -> Result<Self, Error> {
```

Comment:
Not a strong opinion, but curious why you prefer this from a `Config`, rather than a `grouping_power` and `max_value_power`. It seems to me like the most common use of this is going to be rehydrating serialized data in which case it will be stored as the latter.

Reply:
Not a strong preference. I've changed it to take the parameters instead of the config.